File: Assignment No.38/Program5.java
/* Problem statement : Write a program which accept one number from user and on its first 4 bits return modified number.
Input : 73
Output : 79
*/
import java.util.*;

class Program5
{
	public static int OnBit(int iNo)
	{
		int iResult=0;
		int iMask=0x0000000f;

		return (iNo | iMask);
		
		
	}
	public static void main(String args[])
	{
		Scanner sobj=new Scanner(System.in);
		int iNo=0;
		System.out.println("Enter the Number :");
		iNo=sobj.nextInt();

		int iRet=0;

		iRet=OnBit(iNo);
		System.out.println("Modified Number :"+iRet);
	}
}

File: Assignment No 6/Program5.c
//Write a program which accept N and print first 5 multiple of N.
//Input : 4
//Output : 4 8 12 16 20

#include<stdio.h>
void MultipleDisplay(int iNo)
{
	if(iNo < 0)
	{
		iNo=-iNo;
	}
	int i=0;

	for(i=1;i<=5;i++)
	{
		printf("%d\t",iNo*i);
	}
}
int main()
{
	int iValue=0;
	printf("Enter the number :\n");
	scanf("%d",&iValue);
	MultipleDisplay(iValue);
	return 0;
}